source/faucet/main.d

```diff
@@ -366,6 +369,36 @@ public class Faucet : FaucetAPI
             this.client.putTransaction(tx);
         }
     }
+
+    /// Returns: The path at which the files are located
+    private string getStaticFilePath () const
+    {
+        if (std.file.exists("frontend/src/index.html"))
+        {
+            writeln(std.file.getcwd());
+            return std.file.getcwd() ~ "/frontend/src/";
+        }
+        throw new Exception("Files not found. " ~
+                            "This might mean your faucet is not installed correctly. " ~
+                            "Searched for `index.html` in '" ~ std.file.getcwd() ~
+                            "/frontend/src/'.");
+    }
+
+    /// Start listening for requests
+    public void start (URL url)
+    {
+        string path = this.getStaticFilePath();
+        auto settings = new HTTPServerSettings(url.host);
+        settings.port = url.port;
+        auto router = new URLRouter;
+
+        /// Convenience redirect, as users expect that accessing '/' redirect to index.html
+        router.match(HTTPMethod.GET, "/", staticRedirect("/index.html", HTTPStatus.movedPermanently));
+        /// By default, match the underlying files
+        router.match(HTTPMethod.GET, "*", serveStaticFiles(path));
+
+        listenHTTP(settings, router);
+    }
```

Comment:
Wait you shouldn't have 2 routers / `listenHTTP` calls.

Reply:
Should I move the code to the main? There is a `new` router there.

Reply:
Deleted the `start` method and moved the code to `main` due to duplication.